index.js

```diff
@@ -1,7 +1,13 @@
-if (process.env.WFM_USE_MEMORY_STORE === "true") {
-  console.log('Using In-memory Store');
-  module.exports = require('./lib/array-store');
-} else {
-  console.log('Using $fh.db Store');
-  module.exports = require('./lib/fh-db-store');
-}
\ No newline at end of file
+var _ = require('lodash');
+module.exports = function selectStore(config) {
+  config = _.defaults(config, {
+    persistent: process.env.WFM_USE_MEMORY_STORE !== "true"
```

Comment:
I really think we should remove the env var completely.

I think that anything that changes the config of the store should come from the thing consuming it.